src/xGadgetron/pGadgetron/Gadgetron.py

```diff
@@ -388,6 +388,16 @@ def axpby(a, x, b, y):
             (a.real, a.imag, x.handle, b.real, b.imag, y.handle)
         return z;
 
+    def copy(self):
+        '''alias of clone'''
+        return self.clone()
+    def conjugate(self):
+        '''Returns the complex conjugate of the data '''
```

Comment:
seems to be modifying the object itself as well. Is this intentional? If so, needs to be in the docstring

Reply:
I guess that's a mistake. We should return a new object like this
```python
out = self.clone()
out.fill(self.as_array().conjugate())
return out
```